### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by walkerjoe**
Removing unnecessary code

### CODE REVIEW

File: gslib/tests/test_cp.py [hunk 1]
```diff
@@ -495,20 +495,6 @@ def test_noclobber_different_size(self):
       self.assertIn('Skipping existing item: %s' % suri(f), stderr)
       self.assertEqual(f.read(), b'quux')
 
-  @SequentialAndParallelTransfer
```
Comment (catleeball) [inline gslib/tests/test_cp.py]: Sorry about this merge goof, thanks for catching it. :)

Comment (houglum) [inline gslib/tests/test_cp.py]: My test! Don't kill it! ;)

Comment (catleeball) [reply gslib/tests/test_cp.py]: Oh!! Sorry, I thought I'd messed up the merge when we saw there was two of those! Thanks for letting us know that's intentionally there lol :smiley:

Comment (walkerjoe) [reply gslib/tests/test_cp.py]: @houglum - there are intentionally 2 tests called test_noclobber_different_size() ?

Comment (houglum) [reply gslib/tests/test_cp.py]: Derp. I was looking at the master branch (where this wasn't duplicated) and wondering why you removed it -- looks like this test actually got added in twice in the py-six-current branch.  Thanks for catching that :)

**Pull request merged**